### PR TITLE
edited module-service-provider.stub to load module's factories

### PR DESCRIPTION
**Pull request by xfudox**
Ok, new clean pull requests.

### CODE REVIEW

File: Modules/Workshop/Scaffold/Module/stubs/module-service-provider.stub [hunk 1]
```diff
@@ -54,4 +57,16 @@ class $MODULE$ServiceProvider extends ServiceProvider
...
+
+    /**
+     * Register factories.
+     *
+     * @param  string  $path
+     * @return void
+     */
+    protected function registerEloquentFactoriesFrom($path)
+    {
+        debug_print_backtrace();
```
Comment (nWidart) [inline Modules/Workshop/Scaffold/Module/stubs/module-service-provider.stub]: Why is this needed?

Comment (xfudox) [reply Modules/Workshop/Scaffold/Module/stubs/module-service-provider.stub]: F**k, my fault. Fixed.

**Comment by nWidart** [comment]
Thanks!
